proto/devices.proto

```diff
@@ -21,6 +21,8 @@ service DevicesApi {
 message Device {
   string name = 1;
   smartcore.traits.Metadata metadata = 2;
+  // The name of the node that this device is announced from.
+  string node = 3;
```

Comment:
I don't think this change is related and deserves some more thought to implement properly as a "where did it come from" concept. I can't see anything in the rest of the PR that requires this, can you remove it please?

Reply:
Yep, added this by mistake